tests/appsec/test_traces.py

```diff
@@ -328,6 +328,7 @@ def validate_request_headers(span):
 class Test_DistributedTraceInfo:
     """Distributed traces info (Services, URL, trace id)"""
 
+    @irrelevant(True, reason="test non implemented")
```

Comment:
In that case, juste remove the test. `irrelevant` can make think the test is not relevant.

Reply:
test removed.